_summaries/maas2013rectifier_1.md

```diff
@@ -3,17 +3,45 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kelseyball
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Core idea: Rectifier DNNs > Sigmoidal DNNs in Speech Recognition
+
+### Main Contributions
+- This paper demonstrates that deep neural network (DNN) models for speech recognition perform better when they use rectifier activation functions instead of sigmoidal ones.
+- Previous work compares the two, but without isolating all other variables. This paper isolates the effect of the activation function in DNN-based speech recognition.
+- The authors theorize that the performance benefit is attributable to "sparse-disperse" coding properties of Rectifier functions and provide evidence in support of this theory.
+
+### Motivation for Rectifier Functions
+Drawbacks of sigmoidal activation functions:
+- Sigmoidal activation functions suffer from "vanishing gradients" which can slow training and/or cause the network to converge to poor local minima.
+- They do not induce hard-sparsity (values of 0) in activations, which can be a drawback if you want to use activations as feature representations for some downstream task.
+Both of these issues can, in principle, be solved with rectifier activation functions.
+
+<img width="500px" src="maas2013rectifier_1a.png"/>
+
+### Experiment setup
+- Task: Large Vocabulary Continuous Speech Recognition (LVCSR)
```

Comment:
I would mention the type of loss function used and the type of optimization algorithm. Another part of their experiment was varying the number of nonlinear layers they used. I would also include that aspect of their experiment in your experiment setup.